src/components/prescriptions/PrescriptionAddComponent.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+    <div
+        class="prescriptionBox"
+        @submit.prevent="addTest">
+        <h3 class="prescriptionBox__header">Dodaj receptę:</h3>
+        <form class="prescriptionBox__form prescriptionForm">
+            <div>
+                <div class="inputBox">
+                    <label for="nameAndSurnameDoctor">
+                        Imię i nazwisko lekarza:</label
+                    >
+                    <select v-model="nameAndSurnameDoctor">
+                        <option
+                            disabled
+                            value="">
+                            Wybierz lekarza
+                        </option>
+                        <option
+                            v-for="research in getDoctorsArray()"
+                            :key="research.registryId">
+                            {{ research.doctorName }}
+                        </option>
+                    </select>
+                </div>
+                <div class="inputBox">
+                    <label for="medicine">Lek:</label>
+                    <select v-model="medicine">
+                        <option
+                            disabled
+                            value="">
+                            Wybierz lek
+                        </option>
+                        <option
+                            v-for="research in getMedicineArray()"
+                            :key="research.medicinalProductId">
+                            {{ research.medicineName }}
+                        </option>
+                    </select>
+                </div>
+
+                <div class="inputBox">
+                    <label for="frequency">Częstotliwość:</label>
```

Comment:
Same problem as previous. For value must be equal to input's ID. for="frequency" id="frequency"